packages/remix/src/generators/error-boundary/lib/normalize-options.ts

```diff
@@ -0,0 +1,18 @@
+import {joinPathFragments, readProjectConfiguration, type Tree} from '@nx/devkit';
+import type {ErrorBoundarySchema} from "../schema";
+
+
+export function normalizeOptions(tree: Tree, schema: ErrorBoundarySchema): ErrorBoundarySchema {
+  const project = readProjectConfiguration(tree, schema.project);
+
+  const pathToRouteFile = joinPathFragments(project.root, schema.routePath);
+
+  if (!tree.exists(pathToRouteFile)) {
+    throw new Error(`Route file specified does not exist "${pathToRouteFile}". Please ensure you pass a correct path to the file, relative to the project root.`);
+  }
+
```

Comment:
We could optionally do some work here to detect which version should be used rather than needing to manually set it. Example here: https://github.com/nrwl/nx-labs/blob/main/packages/remix/src/generators/meta/lib/normalize-options.ts

Reply:
I don't know if this is a good idea. Someone can be using Remix v1 with future flags enabled